SchleswigGottorf/GotD307.xml

```diff
@@ -17,17 +17,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   Forschungsumgebung / Beta maṣāḥǝft</publisher>
                <availability>
                   <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
-                     <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
-                     </p>
                   </licence>
                </availability>
-               <date>2016-06-07T17:45:12.922+02:00</date>
-            </publicationStmt>
+                </publicationStmt>
             <sourceDesc>
                 <msDesc xml:id="ms">
                     <msIdentifier>
                         <repository ref="INS0405Gottorf_Castle_"/>
-                        <idno>Manuscript D307</idno>
+                        <idno>Schleswig, Archaeological Museum Schloss Gottorf, D 307</idno>
```

Comment:
no this is too long, as the title of the record itself will already contain the name of the institition.
under `idno `only the shelf mark should be, but it is best if it can be clear
I would put
`<idno facs="Gottorf/D307/D307-" n="????">Gottorf HS D 307</idno>` etc

Reply:
I took over the formula Manuscript D ... from Sophia's description which was set up at the TEI training. "Gottorf" is part-information, it should be rather Archaeological Museum Schloss Gottorf. If D should not be extended to "Dettenberg",  what remains? I V. Six's catalogue it is: 
![grafik](https://user-images.githubusercontent.com/18533839/167800371-8d291ae1-b76b-4c9a-9ebc-b3ae714a900e.png)

Reply:
"Archaeological Museum Schloss Gottorf" is the name of the Institution, it will be printed from the link to the repository (if the institution record is correct). It does not belong to the MS record.
In the `<idno>` we need the shelfmark that goes after the name of the institution, but ideally in the form that is more or less understandable by itself.
If D stands for Dettenberg I think it may be extended

Reply:
"D ..." was provided by the institution, I am not sure if we can extend it showing the name of the collector without consulting the Museum.  The owner left Ethiopia in 1967. I think in other similar cases we do not show the collector's name, like in cases of LBK003, MMU001 etc.